src/js/core/img.js

```
@@ -1,225 +1,157 @@
-                    }, noop)
-                };
-
-            },
-
-            events: ['scroll', 'load', 'resize']
-
+    disconnected() {
+        if (this.img) {
+            this.img.onload = '';
         }
+        delete this.img;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)